src/Tools/RepoUtil/RepoUtil.csproj

```diff
@@ -10,7 +10,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>RepoUtil</RootNamespace>
     <AssemblyName>RepoUtil</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <TargetFramework>net46</TargetFramework>
     <RuntimeIdentifiers>win7</RuntimeIdentifiers>
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <TargetFrameworkProfile />
```

Comment:
What other properties can go away? Would like to delete + ban them with BuildBoss. 

I don't have any objections to globs. It was on my "TODO" list once I got this change merged.